src/patternfly/components/Progress/progress.hbs

```diff
@@ -0,0 +1,51 @@
+{{!--   This builds a progress bar. The handlebars parameters below provide options:
+  pf-c-progress--value The percentage of progress 
+  pf-c-progress--description Description to show above the progress bar 
+  pf-c-progress--modifier Other modifiers: pf-m-lg, pf-m-sm
+  pf-c-progress--inside Modifies to have measure shown inside the progress bar
+  pf-c-progress--success Indicates success
+  pf-c-progress--danger Indicates danger/failure
+  pf-c-progress--no-measure Does not output a measure
+  pf-c-progress__id ID for the component
+ --}}
+<div class="pf-c-progress {{pf-c-progress--modifier}}
```

Comment:
can we add a conditional around the modifier as well as reformat the other conditional to avoid the whitespace that is being created in the html output?